workflow/lib/shared/file_utils.py

```diff
@@ -26,3 +35,121 @@ def get_filename(data_location: dict, info_type: str, file_type: str) -> str:
     # Construct filename by combining components with info_type and file_type
     filename = f"{well_str}{tile_str}{cycle_str}__{info_type}.{file_type}"
     return filename
+
+
+def read_stack(filename, copy=True, maxworkers=None, fix_axes=False):
```

Comment:
As above, let's try to deprecate this in the near future. Will take some digging of why we need read_stack, but I am sure you can figure that out. Not a priority for now